src/Core/Checkout/Document/Renderer/AbstractDocumentRenderer.php

```diff
@@ -73,4 +76,40 @@ protected function isAllowIntraCommunityDelivery(array $config, OrderEntity $ord
 
         return $isCompanyTaxFree && $isPartOfEu;
     }
+
+    protected function isValidVat(OrderEntity $order, ValidatorInterface $validator): bool
+    {
+        $customerType = $order->getOrderCustomer()?->getCustomer()?->getAccountType();
+        if ($customerType !== CustomerEntity::ACCOUNT_TYPE_BUSINESS) {
+            return false;
+        }
+
+        $orderDelivery = $order->getDeliveries()?->first();
+        if (!$orderDelivery) {
+            return false;
+        }
+
+        $shippingAddress = $orderDelivery->getShippingOrderAddress();
```

Comment:
Just curious: is the shipping address the right choice here? Alternative could be the billing address right? 🤔

Reply:
Ah I see, it's similar to the method `isAllowIntraCommunityDelivery`